### ISSUE TITLE
Client side messages

### ISSUE THREAD

**Issue by AugustusUKR** [powercord-community/suggestions#412]
### Description
Simply send messages client sided, you would use this plugin via "powercords command manager"

----

 - [x] I checked if this request wasn't already proposed
 - [x] I made sure this request follows Powercord's [plugin guidelines](https://powercord.dev/guidelines)

**Comment by randomdiscorduser** [comment]
Does this mean that you want to send messages without having them actually sent? Just visible on your own device?

**Comment by DracoClaw** [comment]
Isn't that what the `.echo` command is for?

**Comment by AugustusUKR** [comment]
> Does this mean that you want to send messages without having them actually sent? Just visible on your own device?

Yes, they should also act like a normal message (last forever until deleted, pinned, edited etc)
